### PR TITLE
Testing Express: Update instructions to include setting up Jest

### PR DESCRIPTION
**Pull request by tonyissa**
The lesson for testing node/express applications uses Jest but does not make clear that Jest is needed to make the test work in this particular example. I included some instructions to install, setup, and run Jest and to take a look back at the previous lessons if one does not remember how to. I checked for issues regarding this lesson and couldn't find any of them. Also, if someone wants to edit the prose of the new lesson content, then feel free.

## Because

The test function will throw a reference error if Jest is not used.

## This PR

- Clarify language that this test utilizes Jest

- Include instructions that the user must install Jest and setup a npm test script

- Instructs the user to run the test script to see the results of the test

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: nodeJS/testing_express/testing_routes_and_controllers.md [hunk 1]
```diff
@@ -58,11 +58,14 @@ These two files, `app.js` and `index.js` simply define a couple of routes and th
...
 To facilitate actually testing these routes we're going to use a library called [SuperTest](https://github.com/visionmedia/supertest), so go ahead and `npm install supertest --save-dev` and while it's installing take a few minutes to look through the readme on their git repo (linked above).
 
+As well, you will need to install Jest and create a new npm script which runs Jest. If you've forgotten how to do this then this may be a good time to take a look [at the previous Javascript testing lessons](https://www.theodinproject.com/lessons/node-path-javascript-testing-basics).
+
 In the examples below we're going to use SuperTest inside of a Jest style describe/test block, but the syntax and use of these are common among most testing libraries, so the concepts should be easily replicated in Mocha with Chai or Jasmine or Tape or whatever testing library you prefer.
 
 Here's our test file:
 
 ~~~javascript
+//// index.test.js
```
Comment (bycdiaz) [inline nodeJS/testing_express/testing_routes_and_controllers.md:68]: Was this intended to be left here? Just wanna make sure.

**Review by bycdiaz — COMMENTED** [review]
Just one comment. Thanks for this!

File: nodeJS/testing_express/testing_routes_and_controllers.md (hunk 1 above)
Comment (tonyissa) [reply nodeJS/testing_express/testing_routes_and_controllers.md:68]: I did put that there intentionally for clarity but feel free to remove it if you'd like. I saw that style of commenting the file that you are in code blocks from other Odin lessons.

**Comment by bycdiaz** [comment]
@tonyissa Looks like there's a conflict on the branch currently. Ping me here once that conflict has been resolved and we'll get this merged in. Thanks!

**Comment by thatblindgeye** [comment]
@tonyissa checking in on the status of this, have you seen the above comment?

**Pull request closed without merging**